modules/dqt/jsx/react.tabs.js

```diff
@@ -454,9 +467,10 @@ class ViewDataTabPane extends Component {
     ) : null;
 
     let sessionsEmpty = this.props.filter.session.length === 0;
-    if (this.props.AllSessions.length > 0) {
+    if (this.state.sessionsLoaded) {
```

Comment:
@ridz1208 this change is part of the solution for the "Fixes page error when trying to filter on select elements" bug. this.state.sessionsLoaded will be set to true once sessions have been retrieved and so if sessions are emptied then the run query button doesn't get reset depending on if fields are selected.